ibis/backends/base/sql/registry/window.py

```diff
@@ -104,20 +91,20 @@ def cumulative_to_window(translator, op, window):
 
 def time_range_to_range_window(_, window):
     # Check that ORDER BY column is a single time column:
-    order_by_vars = [x.op().args[0] for x in window._order_by]
+    order_by_vars = [x.args[0] for x in window._order_by]
```

Comment:
Do we expect `ops.SortKey` instances here? If yes could we use `x.expr` instead?

I know that the window._order_by member is not properly validated yet.

Reply:
`ops.SortKey`s are allowed, to support `DESC`